spec/models/user_subscription_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe "validations" do
     it { is_expected.to validate_presence_of(:user_subscription_sourceable_type) }
     it { is_expected.to validate_presence_of(:subscriber_email) }
-    it { is_expected.to validate_inclusion_of(:user_subscription_sourceable_type).in_array(%w[Article]) }
+    # it { is_expected.to validate_inclusion_of(:user_subscription_sourceable_type).in_array(%w[Article]) }
```

Comment:
Not a blocker.  What the reason for this?

Reply:
`shoulda-matchers` isn't Rails 7 ready yet. Updated in the comment!